saleor/order/utils.py

```diff
@@ -34,9 +38,11 @@ def decorator(*args, **kwargs):
 
 @receiver(status_changed)
 def order_status_change(sender, instance, **kwargs):
```

Comment:
This function is a signal receiver and it's used in a different way than other functions in this module. Maybe we should move it to a separate file. e.g. `signals.py`?